.github/main.workflow

```diff
@@ -0,0 +1,12 @@
+workflow "New workflow" {
+  on = "push"
+  resolves = ["Hello World"]
+}
+
+action "Hello World" {
+  uses = "./action-a"
```

Comment:
It is unclear whether the Visual Workflow Editor will become available if this referenced action `action-a` is not yet committed to master.